python/kserve/requirements.txt

```diff
@@ -17,7 +17,7 @@ cloudevents>=1.2.0
 avro>=1.11.0
 boto3==1.21.0
 psutil>=5.9.0
-ray[serve]==1.10.0
+ray==2.0.0
```

Comment:
Can we limit to only serve instead of whole Ray package?

Reply:
@yuzisun 
It's possible. I was mistaken.
I limited it to serve and uploaded it again.